src/ng-s3upload/directives/s3-upload.js

```diff
@@ -11,10 +11,8 @@ angular.module('ngS3upload.directives', []).
         $scope.success = false;
         $scope.uploading = false;
 
-        $scope.barClass = function () {
-          return {
-            "bar-success": $scope.attempt && !$scope.uploading && $scope.success
-          };
+        $scope.isUploadSuccessful = function(){
```

Comment:
We'd probably want to allow a dynamic class instead of a static class defined in the template, so maybe barClass is a better approach, using a const that defines the defaultClass and allow each theme to override it.